core/src/main/java/org/apache/iceberg/ManifestEntriesTable.java

```diff
@@ -69,18 +69,35 @@ MetadataTableType metadataTableType() {
 
   private static class EntriesTableScan extends BaseTableScan {
 
-    EntriesTableScan(TableOperations ops, Table table, Schema schema) {
+    private final String scannedTableName;
```

Comment:
I went with `scannedTableName` as opposed to `entriesTableName` to match `StaticScanClass`, as this class could be simplified by extending `StaticTableScan` which uses that name.